### PR TITLE
groups: fix for exposures - path should be the path in the BI

### PR DESCRIPTION
**Pull request by haritamar**

### CODE REVIEW

File: elementary/monitor/api/groups/groups.py [hunk 1]
```diff
@@ -53,6 +53,12 @@ def _update_dbt_group(
             return
 
         artifact_full_path_split = artifact.normalized_full_path.split(posixpath.sep)
+        if isinstance(artifact, NormalizedExposureSchema):
+            # For exposures, we want the path to be the path in the BI
+            artifact_full_path_split = artifact_full_path_split[
+                :2
+            ] + artifact.fqn.split("/")
```
Comment (haritamar) [inline elementary/monitor/api/groups/groups.py:60]: first two parts of the full path are the package name and "exposures", which we want to keep

Comment (NoyaArie) [reply elementary/monitor/api/groups/groups.py:60]: can we add a test?

**Review by NoyaArie — APPROVED** [review]
👍

File: elementary/monitor/api/groups/groups.py (hunk 1 above)
Comment (haritamar) [reply elementary/monitor/api/groups/groups.py:62]: I'll actually replace with more straightforward logic

**Pull request merged**